src/cast_core/src/CollectionScanner.cpp

```diff
@@ -417,6 +417,7 @@ void CollectionScanner::run() {
     std::optional<bsoncxx::types::b_timestamp> readClusterTime;
     for (auto&& config : _loop) {
         for (const auto&& _ : config) {
+            BOOST_LOG_TRIVIAL(info) << "Starting " << this->defaultName() << " execution";
```

Comment:
Do we want debug or info? In https://jira.mongodb.org/browse/TIG-3421 the original request was debug, but then asked to consider info.

Reply:
I think the debug level, since this could be spammy for e.g. 100-threaded workloads. Also if the terminal logging gets backed up it might slow down the workload?

Also, this will only print when a single actor thread is finished, right? don't think it would be very useful if there's multiple CrudActors doing different things, for example. It sounds like we'd want to pipe through the human-given name too.